integration/tests/cook/test_basic.py

```diff
@@ -1868,6 +1868,10 @@ def test_default_container_for_pool(self):
         if len(matching_containers) == 0:
             self.skipTest("No rule matches the default submit pool")
         expected_container = matching_containers[0]
+        # Make sure we have a different configured docker image and default image, we fail the test to make sure that we
+        # don't inadvertently skip it.
+        self.failIfEqual(expected_container["docker"]["image"],util.docker_image())
```

Comment:
Can we run this through the Python code formatter?

Reply:
Done